src/internal/client/apis/AuthenticationServiceApi.ts

```diff
@@ -61,7 +61,7 @@ export class AuthenticationServiceApi extends runtime.BaseAPI {
 
     const response = await this.request(
       {
-        path: `/api/rest/v1/authentication/login`,
+        path: `:5000/api/rest/v1/authentication/login`,
```

Comment:
technically, a port is not considered to be part of the path.

https://www.rfc-editor.org/rfc/rfc1738#section-3.1